ONSdigital/census-rm-pubsub
# Basic component test

## Motivation and Context
Added component test to project

## What has changed
Add Docker compose files and scripts and updated Makefile to run these tests
Added in component test

## How to test?
This should work by cloning the repo, checking out this branch and run the tests:
**_git clone git@github.com:ONSdigital/census-rm-pubsub.git
git checkout basic-component-test
make test_**

These should complete successfully.  'make test' runs the normal tests and then builds the census-rm-pubsub to the local docker registry, then brings up rabbitmq, pubsub_emulator & census-rm-pubsub.  Make test leaves this running

## Links
https://trello.com/c/hnDKKRdj/484-tests-for-running-census-rm-pubsub-13
